WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -23,12 +23,20 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
         return isViewLoaded && view.window != nil
     }
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    init() {
+        super.init(nibName: nil, bundle: nil)
 
+        // register table view cells specific to this controller as early as possible.
```

Comment:
(nit) I would suggest adding a reference to this ticket just to make sure no one moves it back to `viewDidLoad` later.

Reply:
Added in https://github.com/wordpress-mobile/WordPress-iOS/pull/23368/commits/8b08d47d675112a04c7e1a5d0973b91051b5c384 👍